lib/perl/Genome/VariantReporting/Framework/Component/Report/MergedReport.pm

```diff
@@ -540,5 +541,25 @@ sub get_entry_source {
     die sprintf("No entry source for report (%s)", $report);
 }
 
+sub category {
+    my $self = shift;
+
+    my @report_users = map { $_->users('label like' => 'report:%') } $self->report_results;
+    my $category;
+    for my $user (@report_users) {
+        if ($user->label =~ /report:(.*)/) {
+            my $metadata_json = $1;
+            my $m = from_json($metadata_json);
```

Comment:
That is something we should discuss in more detail, I agree (on the face of it) that it seems better to have that information live in a Metric instead of a User label.  

Perhaps the `DelegatesToResult` class could take an argument that would create the metrics in almost the same manner as it takes a `user` and a `label` argument now.

We'll have to think about migration too if we decide to switch it.